File: lab/python/tempo/conversao_segundos_minutos.py
def converter_segundos_em_minutos(segundos):
    return segundos // 60


def converter_segundos(segundos):
    minutos = segundos // 60
    resto = segundos % 60
    return minutos, resto


entrada = int(input())
minutos, resto = converter_segundos(entrada)
print(f"{minutos} minuto(s) e {resto} segundo(s)")

entrada = int(input())
saida = converter_segundos_em_minutos(entrada)
print(saida)

File: lab/javascript/tempo/conversao_segundos_minutos.js
function converter_segundos_em_minutos(segundos){
    return Math.floor(segundos / 60);
}

function converter_segundos(segundos){
    let minutos = Math.floor(segundos / 60);
    let resto = Math.floor(segundos % 60);
    return [minutos, resto];
}


const prompt = require('prompt-sync')();
let entrada = parseInt(prompt());
let [minutos, resto] = converter_segundos(entrada);
console.log(`${minutos} minuto(s) e ${resto} segundo(s)`);

entrada = parseInt(prompt());
saida = converter_segundos_em_minutos(entrada);
console.log(saida);
